network-api/networkapi/urls.py

```diff
@@ -41,14 +41,12 @@
     url(r'^wagtail/', include(wagtail_urls))
     if settings.ENABLE_WAGTAIL else None,
 
+    url(r'^fellowship/(?P<path>.*)', RedirectView.as_view(
+        url='/fellowships/%(path)s',
+        query_string=True
+    )),
     url(r'^soc/', include('social_django.urls', namespace='social')),
```

Comment:
please drop this line too, it's a dupe of line 33

Reply:
weird, thought i deleted it 👀